app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,9 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <%= csrf_meta_tags %>
   <%= csp_meta_tag %>
+  <% if Rails.application.secrets.esbuild_devserver || !Rails.env.local? %>
+    <%= stylesheet_link_tag "application", media: "all" %>
+  <% end %>
```

Comment:
Remove both the condition and move it inside `_bundle.html.erb` together with the application bundle for ESBuild.